addons/cert-manager/README.md

```diff
@@ -0,0 +1,62 @@
+# Certification Manager Helm Chart
+
+Certification Manager is a Kubernetes addon to automate the management and issuance of TLS certificates from various issuing sources.
+It will ensure certificates are valid and up to date periodically, and attempt to renew certificates at an appropriate time before expiry..
+
+## Installation
+Below terraform script shows how to use Certification Manager Terraform Addon, A complete example is also given [here](https://github.com/clouddrove/terraform-helm-eks-addons/blob/master/_examples/complete/main.tf).
+```hcl
+module "addons" {
+  source  = "clouddrove/eks-addons/aws"
+  version = "1.0.0"
+  
+  depends_on       = [module.eks.cluster_id]
+  eks_cluster_name = module.eks.cluster_name
+
+  certification_manager   = true
```

Comment:
- `certification_manager` will be available to use in next release. Please update version to `"0.1.0"`
- also update `depends_on = [module.eks]`